akka-javasdk/src/main/java/akka/javasdk/annotations/AgentDescription.java

```diff
@@ -19,4 +19,5 @@
 public @interface AgentDescription {
   String name();
   String description();
+  String role() default "";
```

Comment:
for simplicity this is only supporting one role per agent, but we could make it an array if you think that would be needed?